packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -24,10 +24,17 @@ enum _AnimationDirection {
 
 /// A controller for an animation.
 ///
-/// An animation controller can drive an animation forward or backward and can
-/// set the animation to a particular value. The controller also defines the
-/// bounds of the animation and can drive an animation using a physics
-/// simulation.
+/// This class lets you perform tasks such as:
+///
+/// * Play an animation forward or backward, or stop and resume an animation.
+/// * Set the animation to a specific value.
+/// * Define the upper and lower bounds of an animation.
+/// * Create animation effects such as a fling, by using a physics simulation.
```

Comment:
ideally, link from these points to the actual members. For example, `... [forward] or in [reverse], or [stop] and [resume] an...`